catalyst_voices/lib/widgets/tooltips/voices_plain_tooltip.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:flutter/material.dart';
+
+/// [VoicesPlainTooltip]
```

Comment:
Probably this line could be removed, doesn't add anything relevant

Reply:
You're right, will remove